table/tables/virtual_tables.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
Do we need a test file for virtual_tables?

Reply:
I think we should to add a mysql test case for this file instead of adding a test file, It is more light.